### DISCUSSION TITLE
Asked for Steam Guard code every time

### DISCUSSION THREAD

**Discussion by lennier1** [SteamRE/DepotDownloader#438] [Q&A]
I have Steam Guard (2FA) enabled on my account. I believe that when I used DepotDownloader in the past, it would ask for a Steam Guard code the first time, but for at least awhile after that, Steam would remember me and not ask again. But now, it's asking every single time I download something, even when I run it again from the same Powershell window without changing IP and without any significant delay. Is this a security change Steam has made? Is there a way to make it remember me? Would this be considered a bug?

**Comment by yaakov-h** [comment]
Are you using v2.5.0, or an older version?

**Reply by lennier1** [reply]
I'm using v2.5.0. It looks like the last time I used it, when it wasn't asking every time, I was using v2.4.7.

**Reply by yaakov-h** [reply]
Hmm. Can you provide the exact arguments/parameters that you're passing? Redact your username and password, of course.

**Reply by lennier1** [reply]
Here's an example:

DepotDownloader.exe -app 2334160 -all-platforms -all-languages -user [redacted] -pass [redacted]

**Reply by yaakov-h** [reply]
Try adding `-remember-password`
I believe this is related to the recent server changes related to #401.

**Reply by lennier1** [reply]
You just use -remember-password instead of -pass? I'm seeing a lot of login problems doing that. This did work, and didn't ask for a code:

./DepotDownloader.exe -app 2385060 -all-platforms -all-languages -user [redacted] -remember-password [redacted]

But then I tried several that didn't work (not sure how consistent this is):

./DepotDownloader.exe -app 2420130 -all-platforms -all-languages -user [redacted] -remember-password [redacted]
Connecting to Steam3... Done!
Logging [redacted] into Steam3...
Unable to login to Steam3: AccessDenied
Disconnected from Steam
Unable to get steam3 credentials.
Error: InitializeSteam failed
